www/artworks/post.php

```diff
@@ -31,6 +52,17 @@
 		throw new Exceptions\InvalidCaptchaException();
 	}
 
+	$uploadError = $_FILES['color-upload']['error'];
+	if ($uploadError > 0){
+		// see https://www.php.net/manual/en/features.file-upload.errors.php
+		$message = match ($uploadError){
+			1 => 'Image upload too large (maximum ' . ini_get('upload_max_filesize') . ')',
```

Comment:
Replace the `1` with `UPLOAD_ERR_INI_SIZE` and remove the comment above with the link to php.net.

```
UPLOAD_ERR_INI_SIZE => 'Image upload too large (maximum ' . ini_get('upload_max_filesize') . ')',
```

Reply:
(I suppose you can keep the comment above that links to php.net if you'd like. The point was to use the `UPLOAD_ERR_INI_SIZE` constant directly and not depend on the link to know what the `1` meant.)